assets/src/edit-story/components/canvas/displayLayer.js

```diff
@@ -56,6 +56,7 @@ function DisplayLayer() {
       <DisplayPageArea
         ref={setPageContainer}
         fullbleedRef={setFullbleedContainer}
+        background={currentPage?.backgroundColor}
```

Comment:
But aren't all the background properties applied in `PageArea` overwritten by the CSS rules in `DisplayPageArea`?

Hm, now that I look into it, `PageArea` doesn't accept (nor use) a `className` property so all the CSS defined in `DisplayPageArea` is ignored, isn't in?

Reply:
Hmm, looks like generally there were some unused attributes in other places as well. Removed in ce33b4c.